vaas-app/src/vaas/cluster/cluster.py

```diff
@@ -148,12 +146,17 @@ class ParallelLoader(ParallelExecutor):
     def __init__(self, max_workers=10):
         ParallelExecutor.__init__(self, max_workers)
         self.api_provider = VarnishApiProvider()
+        self.loader_handler = VclLoader
 
     def _append_vcl(self, vcl, server, future_results, executor):
         """Suppress exceptions if cannot load vcl for server in maintenance state"""
-        loader = VclLoader(self.api_provider.get_api(server), server.status == 'maintenance')
+        loader = self.loader_handler(self.api_provider.get_api(server), server.status == 'maintenance')
```

Comment:
You just assing class name to variable (#149) - creating objects is still duplicated